regression/basic_flow.yaml

```diff
@@ -0,0 +1,32 @@
+name: streaming-regression-test-basic-flow
+compute:
+  cluster: r1z1
+gpu_num: 8
+command: |-
+  cd streaming
+  composer -n 1 regression/dataset.py --create --compression gz
+  composer -n 1 regression/dataset.py --delete
```

Comment:
Do you know what would happen if anyone of the command fails ? will the job terminate or it will go onto next command to execute ?

Reply:
Job will terminate